pages/report/report.yaml

```diff
@@ -0,0 +1,74 @@
+id: report
+type: PageHeaderMenu
+properties:
+  title: Report
+layout:
+  contentGutter: 16 # Set a gutter of 16px between all the cards on the page
+requests:
+  # Requests are referenced from their own yaml files in the requests folder
+  - _ref: pages/report/requests/get_scores_by_genre.yaml
+  - _ref: pages/report/requests/get_top_100_score_difference_movies.yaml
+
+events:
+  # A list of actions that gets completed when this page is first loaded.
+  onInitAsync:
```

Comment:
@StephanieJKS Maybe we should change to `onMountAsync` and add a loading state to the charts?